test/test_odm2_dao_sqlite.py

```diff
@@ -5,7 +5,8 @@
 
 from wof.examples.flask.odm2.timeseries.odm2_timeseries_dao import Odm2Dao as OdmDao  # noqa
 
-ODM2_DATABASE_URI = 'sqlite:///' + "./test/odm2/ODM2.sqlite"
+# ODM2_DATABASE_URI = 'sqlite:///' + "./test/odm2/ODM2.sqlite"
```

Comment:
@ocefpaf Ah, yes. Will do. Thanks. Looks like the changes in the dao passes now! 😄